src/sas/qtgui/MainWindow/MainWindow.py

```diff
@@ -96,14 +94,17 @@ def run_sasview():
 
     # DO NOT move the following import to the top!
     # (unless you know what you're doing)
-    import qt5reactor
+    from sas.qtgui.Utilities import ReactorCore
     # Using the Qt5 reactor wrapper from https://github.com/ghtdak/qtreactor
-    qt5reactor.install()
+    ReactorCore.install()
 
     # DO NOT move the following import to the top!
     from twisted.internet import reactor
 
-    screen_resolution = app.desktop().screenGeometry()
+    screen = QGuiApplication.screenAt(QCursor.pos())
+    screen_resolution = screen.geometry().center()
+
+    # screen_resolution = app.primaryScreen().screenGeometry()
 
     # Show the main SV window
     mainwindow = MainSasViewWindow(screen_resolution)
```

Comment:
As a follow-up, removing these lines and passing `None` to the main sasview window resolves the error I am seeing. `screen_resolution` is no longer being used in the `MainSasViewWindow` and could probably be removed.

```suggestion
    mainwindow = MainSasViewWindow(None)
```